### PR TITLE
Docs misc mac 3

### PR DESCRIPTION
**Pull request by smoe**

### CODE REVIEW

**Comment by smoe** [comment]
@hansu - reapplied everything against 2.9.

**Comment by smoe** [comment]
Rebase to 2.9 and push to trigger rebuild.

File: src/hal/components/plasmac.comp [hunk 1]
```diff
@@ -55,12 +55,12 @@ pin in  bit     cut_recovery                "recover from cut error";
 pin in  float   cut_volts                   "cut voltage (volts)";
 pin in  bit     cutting_start               "start a new cut, connect to spindle.0.on";
 pin in  bit     debug_print                 "if true will print state changes as a debug aid";
-pin in  bit     external_estop              "external E-stop input";
+pin in  bit     external_estop              "external estop input";
```
Comment (hansu) [inline src/hal/components/plasmac.comp:58]:
```suggestion
pin in  bit     external_estop              "external E-stop input";
```
I would keep "E-stop".

File: docs/src/user/user-intro.adoc [hunk 2]
```diff
@@ -125,9 +122,9 @@ GMOCCAPY:: <<cha:gmoccapy,GMOCCAPY>>, a touch screen GUI based on Gscreen. GMOCC
 .GMOCCAPY, a touch screen GUI based on Gscreen
 image::../gui/images/gmoccapy_3_axis.png["GMOCCAPY, a touch screen GUI based on Gscreen",align="center"]
 
-NGCGUI:: <<cha:ngcgui,NGCGUI>>, a subroutine GUI that provides wizard-style programming of G code. NGCGUI may be
-  run as a standalone program or embedded into another GUI as a series of tabs. The following screen shot
-  shows NGCGUI embedded into Axis:
+NGCGUI:: <<cha:ngcgui,NGCGUI>>, a subroutine GUI that provides wizard-style programming of G code.
+  NGCGUI may be run as a standalone program or embedded into another GUI as a series of tabs.
+  The following screen shot shows NGCGUI embedded into AXIS:
```
Comment (hansu) [inline docs/src/user/user-intro.adoc:127]:
```suggestion
  The following screenshot shows NGCGUI embedded into AXIS:
```

Comment (smoe) [reply docs/src/user/user-intro.adoc]: Addressed in rebase to 2.9.

File: src/hal/components/plasmac.comp (hunk 1 above)
Comment (smoe) [reply src/hal/components/plasmac.comp]: Thanks, yes!

**Comment by smoe** [comment]
@hansu - done.

**Comment by smoe** [comment]
@hansu - conflicts resolved.

**Pull request merged**